roles/dhcp/tasks/dhcp.yml

```diff
@@ -14,3 +14,23 @@
     src: '{{ dhcp_config_temp_loc }}'
     dest: '{{ dhcp_config_dest_file }}'
   notify: 'reload dhcp'
+
+- name: 'Disable dhcp services'
+  service:
+    name: '{{ item }}'
+    enabled: no
+    state: stopped
+  with_items:
+  - dhcpd
+  when: 
+  - dhcp_service_enabled == False
```

Comment:
this should read `dhcp_service_enabled|default(True) == False` just in case `dhcp_service_enabled` isn't defined